### PR TITLE
added ctc decoder builds tasks

### LINKED ISSUE #2717: Upload CTC Decoder .whl to GitHub for Releases

### PR DESCRIPTION
**Pull request by imskr**
Added the ctc decoder builds tasks in the python list of the github upload task.
Fixes #2717

### CODE REVIEW

File: taskcluster/scriptworker-task-github.yml [hunk 1]
```diff
@@ -25,9 +25,11 @@ build:
   artifacts_deps:
     python:
       - "darwin-amd64-cpu-opt"
+      - "darwin-amd64-ctc-opt"
```
Comment (lissyx) [inline taskcluster/scriptworker-task-github.yml]: This is good but not enough: i've indicated that you also need to add those in the list of `dependencies` above.

Comment (imskr) [reply taskcluster/scriptworker-task-github.yml]: I will add it

**Comment by imskr** [comment]
Thank you @lissyx , I have added the changes as requested!

**Comment by lissyx** [comment]
That looks good, thanks!

**Pull request merged**